Update Cloudflare-Side-Note.md (#64)
louislam/uptime-kuma-wiki: Cloudflare-Side-Note.md

By default, Cloudflare is not API friendly including Uptime Kuma. Cloudflare may block requests from Uptime Kuma.

> You need to disable or bypass "Browser Integrity Check" in Cloudflare Dashboard.
> 
> At least three methods exist to bypass the Browser Integrity Check:

Reviewer: I think we can shorten this one

```suggestion
You need to disable or bypass "Browser Integrity Check" in Cloudflare Dashboard via one of these methods:
```

> -(Easiest) Add your Uptime Kuma host IP address to IP Access rules as an Allowed address, optionally across every domain in your Cloudflare account: https://developers.cloudflare.com/waf/tools/ip-access-rules/
> -Allow Uptime Kuma to bypass the check via WAF Custom Rules, only applies to one domain at a time: https://developers.cloudflare.com/waf/custom-rules/skip/
> -Use a Configuration Rule to disable the check for your Uptime Kuma IP address: https://developers.cloudflare.com/rules/configuration-rules/

Reviewer: Without these spaces, this is not considered a list

nit (feel fee to overrule): I also changed the links to be text-links as this way the list is better parsable

```suggestion
- (Easiest) Add your Uptime Kuma host IP address to [IP Access rules](https://developers.cloudflare.com/waf/tools/ip-access-rules/) as an Allowed address, optionally across every domain in your Cloudflare account
- Allow Uptime Kuma to [bypass the check via WAF Custom Rules](https://developers.cloudflare.com/waf/custom-rules/skip/), only applies to one domain at a time
- Use a [Configuration Rule](https://developers.cloudflare.com/rules/configuration-rules/) to disable the check for your Uptime Kuma IP address
```

Author: I absolutely forgot to preview my changes to make sure markdown was working. Thanks!
